clothcalc_placeholder.js

```diff
@@ -8806,10 +8808,11 @@
                                         }
                                     })
                         }
-                        t.el
-                            .find(".questRequirementHelp")
-                            .append(
-                                '<a target="_blank" title="#QUESTWIKI#" href="' + Script.protocol + '://' + Script.url + "/quest_redirect.php?id=" + t.id + '" style="margin-left:10px;"><img src="' + Images.questwiki + '" /></a>')
+                        //only old/exported questlines
+                        if (t.group < 142 && (t.group != 69 || t.id >= 20000) || [200,201].indexOf(t.group) > -1) {
```

Comment:
Hmm, so if a miracle happens and someone will start updating our quest database, we will need to keep updating this hardcoded list? Is this really needed?

Reply:
I don't think this will happen anymore. TW-Calc has all Quests imported.
It would be just less confusing for the users.
But you decide.